client/app/payment/paymentFact.js

```diff
@@ -0,0 +1,22 @@
+angular
+  .module('app.payment')
+  .factory('Payment', Payment);
+
+// Dependency injection. Done this way for minification purposes.
+Payment.$inject = ['$http'];
+
+function Payment($http) {
+  return {
+    // GETs user profile info from our MongoDB
+    submitPayment: function (token) {
+      return $http({
+          method: 'GET',
+          url: '/api/charge/',
+          data: token,
```

Comment:
That was a mistake. Didn't mean to commit that file. I'm going to send the post request in the premium factory. Thanks for catching that.